app/controllers/sessions_controller.rb

```diff
@@ -129,19 +129,21 @@ def help
 
   # Omniauth failure endpoint
   def failure
-    security_log :sign_in_failed, reason: params[:message]
```

Comment:
Moved to the CustomIdentity strategy. Basically CustomIdentity redirects you here, but a bot could just not follow the redirect and prevent the log from being created. So I moved it to when the password is actually checked.